irc/client.py

```diff
@@ -393,6 +393,7 @@ class ServerConnection(Connection):
 
     def __init__(self, reactor):
         super(ServerConnection, self).__init__(reactor)
+        self.outgoing_encoding = 'utf-8'
```

Comment:
Since this behavior is repeated in another `Connection` subclass, the behavior should probably exist in the superclass.

I'd prefer the name `transmit_encoding`.